packages/next/src/config/withHeadstartWPConfig.ts

```diff
@@ -186,12 +186,22 @@ export function withHeadstartWPConfig(
 		async rewrites() {
 			const rewrites =
 				typeof nextConfig.rewrites === 'function' ? await nextConfig.rewrites() : [];
+			const topLevelShouldRewriteYoastSEOUrls =
+				headlessConfig?.integrations?.yoastSEO?.enable === true ? 1 : 0;
 
 			sites.forEach((site) => {
```

Comment:
elseewhere in the codebase we use [getSite](https://github.com/10up/headstartwp/blob/develop/packages/core/src/utils/config.ts#L121C17-L121C24) which does proper merging, so you can simply call it before using `site` or call it when building the `sites` array. That way the `site` object will inherit anything from top-level configs and we keep things consistent witht he rest of the codebase.